STUDIO/org.openl.rules.webstudio/webapp/pages/modules/administration/users/addUser.xhtml

```diff
@@ -121,9 +121,9 @@
                         <tbody>
                         <tr>
                             <td id="user-groups">
-                                <span class="width-33pct">
+                                <span class="width-33pct" style="{userGroupStyle}">

```

Comment:
Is it possible to use CSS class instead of inline style?
Mixing content and design makes (re)design less flexible and harder to support.
I would avoid using inline styles if it's possible.

Reply:
Unfortunately, this page is fully done using this style, requires a huge refactoring which is out of scope of this future